dali/dfu/dfurun.cpp

```diff
@@ -54,10 +54,6 @@ test multiclusteradd with replicate
 
 extern ILogMsgHandler * fileMsgHandler;
 
-extern void doKeyDiff(IFileDescriptor *oldf,IFileDescriptor *newf,IFileDescriptor *patchf); // patchf out
```

Comment:
Keydiff/keypatch are not same as dkc are they?

Reply:
I think the dfu functions were implemented with it - see the deleted file.  @jakesmith is that right?
Pushed again because failed to check in a couple of changes.

Reply:
I wouldn't have known, but yes it looks that way.